lib/models/event.js

```diff
@@ -142,5 +145,13 @@ module.exports.MatrixEvent.prototype = {
      */
     isState: function() {
         return this.event.state_key !== undefined;
+    },
+
+    getPushActions: function(client) {
```

Comment:
> shift any more complex stuff like this out to MatrixClient.getPushActionsForEvent(event) - what do you think?

I really like this. I am currently doing persistent storage and am having to construct `MatrixEvents` (the storage has no concept of the `MatrixClient`), so I prefer to keep that class as pure data.